backend/src/main/java/com/github/mysterix5/capstone/textHandling/TextController.java

```diff
@@ -22,4 +26,19 @@ public ResponseEntity<List<WordResponseDTO>> onSubmittedText(@RequestBody TextSu
         log.info("Text in submit text: " + textSubmitDTO.getText());
         return ResponseEntity.ok().body(textService.onSubmittedText(textSubmitDTO.getText()));
     }
+
+    @PutMapping("/getaudio")
```

Comment:
I think it's strange that `getaudio` is a `PutMapping`. I would use `audio` as URL and a `PostMapping` because you are creating an audio.